web/components/application/sources/index.js

```diff
@@ -0,0 +1,10 @@
+export const sources = [
+  'MLH',
+  'Social media',
+  'Website',
+  'Word of mouth',
+  'Club newsletter',
+  'Faculty newsletter',
+  'Prof/in class',
+  'Other',
```

Comment:
Sorry one more thing - I don't think the `Other` option is necessary because the dropdown component already allows the user to enter a new option, and that would be considered "Other"

Reply:
Makes sense, fixed